src/Stryker.Core/Stryker.Core.UnitTest/TestRunners/VsTestRunnersTest.cs

```diff
@@ -123,7 +122,7 @@ private static void DiscoverTests(ITestDiscoveryEventsHandler discoveryEventsHan
                 ContinueWith((_, u) => discoveryEventsHandler.HandleDiscoveryComplete((int)u, null, aborted), tests.Count);
         }
 
-        private TestCase BuildCase(string name) => new TestCase(name, _executorUri, _testAssemblyPath) { Id = new Guid() };
+        private TestCase BuildCase(string name) => new TestCase(name, _executorUri, _testAssemblyPath) { Id = Guid.NewGuid() };
```

Comment:
Just passing by here but still - this produces a random guid instead of the default one, is this change on purpose?

Reply:
It wasn't my intention to break current behavior. I only tried quality refactoring.

My IDE's Sonarlint extention flags `new Guid()` with [the rule S4581](https://rules.sonarsource.com/csharp/RSPEC-4581): _"new Guid()" should not be used_.

However, [it doesn't seem to be reported by Sonarcloud](https://sonarcloud.io/code?branch=v1.0&id=stryker-net&selected=stryker-net%3Asrc%2FStryker.Core%2FStryker.Core.UnitTest%2FTestRunners%2FVsTestRunnersTest.cs). Maybe that rule was disabled on purpose?

Anyway, the tests seem to still pass with the change. Should we keep it or revert back to `new Guid()`?

Reply:
Well I looked at the link, it recommends
```
An empty GUID, in which case Guid.Empty is clearer.
A randomly-generated GUID, in which case Guid.NewGuid() should be used.
A new GUID with a specific initialization, in which case the initialization parameter is missing.
```
So if you want to make an identical change use `Guid.Empty`. At the same time, as long as tests pass, you can probably use whatever :)

Reply:
Looks like it was reverted already. I don't think it matters in any case. We can fix this up again some other time 👌